mlir/utils/jenkins/Dockerfile.rocm

```diff
@@ -65,6 +65,9 @@ RUN ln -s /usr/bin/clang-10 /usr/bin/clang;\
 
 # --------------------- Section 2: MIOpen dialect setups -----------------
 
+# Need "render" group because some CI hosts have /dev/kfd under it.
+RUN groupadd -g 109 render
```

Comment:
[Question] What does 109 mean in the groupadd command?

Reply:
It's the group ID.  The render group happens to be 109 on the machines I checked, and I wanted it to match in the image.

Reply:
What if a different machine happen to have a different group id than 109? We may not want to target the fix for one specific machine.

Reply:
If it's really an ubuntu 20 thing, that shouldn't arise.  However, if we have to accommodate it, we'd need to add code to our image to run at startup time, that ascertains the group of /dev/kfd and adds the current user to it.  Something like

  `sudo usermod -aG `ls -n /dev/kfd | cut -d' ' -f4` $USER`

assuming we can do sudo there, and possibly a newgrp or su, though I don't know how to add startup code to a docker invocation.